peekingduck/configs/draw/bbox.yml

```
@@ -1,5 +1,4 @@
 input: ["bboxes", "img"]
```

It would be more consistent to include this?

```suggestion
input: ["bboxes", "img", "bbox_labels"]
```

---

See below discussion regarding this issue

---

Following below discuss, add bbox labels